packages/config/config/devices/0x0346/keypad.json

```diff
@@ -0,0 +1,387 @@
+// Ring Keypad
+// Param information taken from https://support.ring.com/hc/en-us/article_attachments/360051851012/Keypad_Zwave_UK.pdf
+{
+	"manufacturer": "Ring",
```

Comment:
It may but make sure this matches other Ring devices exactly or there will be duplicate entries on the website.

Reply:
It matches all files in this directory.